### PR TITLE
Cleaned up pull

### PR DESCRIPTION
**Pull request by nigelgbanks**
Replaced hardcoded url's with variable calls.

### CODE REVIEW

File: drupal/islandora/include/admin.form.inc [hunk 1]
```diff
@@ -59,3 +67,26 @@ function islandora_config_form_validate_stomp_url(array $element, array &$form_s
...
+ * @param array $form_state
+ *   Drupal form state.
+ * @param array $form
+ *   Drupal renderable array.
+ */
+function islandora_config_form_validate_karaf_url(array $element, array &$form_state, array $form) {
+  // Try to connect to the server.
+  $url = $element['#value'] . '/cxf';
+  $handle = curl_init($url);
+  curl_setopt($handle,  CURLOPT_RETURNTRANSFER, TRUE);
```
Comment (whikloj) [inline drupal/islandora/include/admin.form.inc]: Travis is failing due to 2 spaces between ```$handle,``` and ```CURLOPT_RETURNTRANSFER```.

**Comment by whikloj** [comment]
@nigelgbanks why did you close the other pull request (https://github.com/Islandora-CLAW/CLAW/pull/166)?

**Comment by nigelgbanks** [comment]
Just to clean up.

**Comment by whikloj** [comment]
@nigelgbanks oh ok, if you want we can always squash your commits before merging it in. Your call.

**Comment by whikloj** [comment]
Testing now...

**Comment by whikloj** [comment]
Sorry for the delay, didn't get finished before the end of the day. Tested and this works for me. Merging in an hour if I don't hear from anyone.

**Comment by DiegoPino** [comment]
:+1: looks fine. You are good to go and you can merge @whikloj

**Pull request merged**